pkg/genesis/vesting_accounts.go

```diff
@@ -68,8 +69,9 @@ func NewGenesisPeriodicVestingAccount(
 
 	balance = banktypes.Balance{
 		Address: address,
-		Coins:   initUnlockedCoins.Add(vestingBalance...),
+		Coins:   vestingBalance,
 	}
+	vestingBalance = vestingBalance.Sub(initUnlockedCoins...)
```

Comment:
[Suggestion] Since this change modifies how the function worked before, it's a good idea to note in the function description that the vesting account balance is now calculated by subtracting `initUnlockedCoins` from `vestingBalance`.